models/tridimensional/docking_validation/generate_pams_3dna.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+import errno
+import math
+import os
+import subprocess
+import sys
+
+from constants import PAM_TEMPLATE_SEQUENCE, DNA_ALPHABET
+from utility import pam_string_from_int
+
+
+def mutate_pam(pam, template_pdb, output_dir):
```

Comment:
I don't think this should be called mutate_pam -- need something that says "pam variant" and "3dna"